app/types/helpers/map.types.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+
+export type Coordinates = {
```

Comment:
I’m not sure, I don’t want this to transform into the constants nightmare in fw desktop.

What do you think of having a `common.types` file?

These types aren’t helpers specific, also used in some components.

Reply:
This name sounds better, I'll move to it.